contracts/JBFundingCycleStore.sol

```diff
@@ -254,13 +264,19 @@ contract JBFundingCycleStore is JBControllerUtility, IJBFundingCycleStore {
     uint256 _metadata
   ) external override onlyController(_projectId) returns (JBFundingCycle memory) {
     // Duration must fit in a uint64, and must be greater than 1000 seconds to prevent manipulative miner behavior.
-    require(_data.duration <= type(uint64).max && _data.duration > 1000, '0x15: BAD_DURATION');
+    if (_data.duration > type(uint64).max || _data.duration <= 1000) {
+      revert INVALID_DURATION();
+    }
 
     // Discount rate token must be less than or equal to 100%. A value of 1000000001 means non-recurring.
-    require(_data.discountRate <= 1000000001, '0x16: BAD_DISCOUNT_RATE');
+    if (_data.discountRate > 1000000001) {
```

Comment:
Declare constant. Though, I think @mejango did this in a recent change. Try merging in `main`.